src/main/java/com/onehouse/metadata_extractor/TimelineCommitInstantsUploader.java

```diff
@@ -251,9 +270,13 @@ private CompletableFuture<Checkpoint> uploadInstantsInSequentialBatches(
           Lists.partition(
               filesToUpload, getUploadBatchSize(CommitTimelineType.COMMIT_TIMELINE_TYPE_ARCHIVED));
     } else {
-      batches =
+      Pair<Checkpoint, List<List<File>>> checkpointBatchesPair =
```

Comment:
If I am understanding correctly, on encountering an incomplete commit we dont process the subsequent commits in the current run but in the next extractor run it gets processed?